exercises/grade-school/GradeSchoolTest.cs

```diff
@@ -1,61 +1,135 @@
+using GradeSchool;
```

Comment:
When deleting the using. I see this error message : 

  Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0246	The type or namespace name 'School' could not be found (are you missing a using directive or an assembly reference?)	TestGradeSchool	C:\Users\rwobb\Documents\Visual Studio 2017\Projects\Challenges\exercism.io\GradeSchool\TestGradeSchool\GradeSchoolTest.cs	9	Active

so when deleting the using it cannot find the add method 
so we need the using or change every school.add to GradeSchool.school.add